### ISSUE TITLE
tslint only shows [SyntaxError: Unexpected token ...] .No indication of which file or line number for error . 

### ISSUE THREAD

**Issue by altanai** [palantir/tslint#3500]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.6.1
- __Running TSLint via__:  CLI

#### TypeScript code being linted

```ts
// code snippet
```

with `tslint.json` configuration:

```json
{
    "extends": [
        "tslint:recommended",
        "tslint-sonarts",
        "tslint-config-prettier"
    ],
    "rules": {
        "await-promise": true,
        "interface-name": [false],
        "max-classes-per-file": [false],
        "no-unused-expression": true,
        "no-unused-variable": true,
        "object-literal-shorthand": false,
        "no-inconsistent-return": true
    }
}
```

#### Actual behavior
[SyntaxError: Unexpected token ...]

does not say which line number or file . No verbose option available too .

#### Expected behavior
show tslint error or warnings with line number

**Comment by ajafff** [comment]
Can you please provide the complete error output?
This looks like either your config contains invalid JSON or one of the rules uses syntax not supported by the version of node you use.
What node version are you using?
Can you reprocude the error without `tslint-sonarts` and `tslint-config-prettier`?

**Comment by altanai** [comment]
Hello Ajafff, i tried removing "tslint-sonarts" and "tslint-config-prettier". Tslint now works as expected . However i do not understand why would it works for others who are running the full tsconfig.json file . 

My node version is 4.5.0 . Cannot update this due to code dependency .

**Comment by ajafff** [comment]
> i tried removing "tslint-sonarts" and "tslint-config-prettier". Tslint now works as expected . However i do not understand why would it works for others who are running the full tsconfig.json file .

As I said earlier, this is probably caused by one of those two packages (or a dependency thereof) using syntax your version of node does not understand - ES6 for example.

And there we have it: https://github.com/SonarSource/SonarTS/blob/master/sonarts-core/tsconfig.json#L3

**Comment by altanai** [comment]
ok thanks . Understood . Closing the issue as resolved
